torch/onnx/__init__.py

```diff
@@ -289,7 +289,9 @@ def forward(self, x):
             This is required for models with large weights that exceed the ONNX file size limit (2GB).
             When False, the weights are saved in the ONNX file with the model architecture.
         dynamic_shapes: A dictionary of dynamic shapes for the model inputs. Refer to
-            :func:`torch.export.export` for more details.
+            :func:`torch.export.export` for more details. This is only used when dynamo is True.
+            Only one parameter `dynamic_axes` or `dynamic_shapes` should be set
+            at the same time.
```

Comment:
Thanks! I was meaning to improve this one. Would it be possible to include a note:

```suggestion
            :func:`torch.export.export` for more details. This is only used (and preferred) when dynamo is True.
            Only one parameter `dynamic_axes` or `dynamic_shapes` should be set
            at the same time.
```

Reply:
For dynamic_axes: 

```
[...] When dynamo=True, a best effort conversion is done to convert to the dynamic_shapes argument for  :func:`torch.export.export`.
```